### PR TITLE
Add gatekeeper operator namespace selection test

### PR DESCRIPTION
**Pull request by marzianor9**

### CODE REVIEW

**Comment by marzianor9** [comment]
Waiting for: https://github.com/openshift-kni/cnf-features-deploy/pull/461 to configure the operator.

File: cnf-tests/testsuites/e2esuite/gatekeeper/gatekeeper.go [hunk 1]
```diff
@@ -55,616 +50,702 @@ var _ = Describe("gatekeeper", func() {
...
+						},
+					},
+				},
+			}
+			err = client.Create(context.Background(), podMutation)
+			Expect(err).NotTo(HaveOccurred())
+
+			// Create namespaces
+			By("Creating the test namespaces")
+			selectedTestNamespace := &corev1.Namespace{
```
Comment (mmirecki) [inline cnf-tests/testsuites/e2esuite/gatekeeper/gatekeeper.go:709]: Can you please update the ns names to refelect that one's resources will be mutated?
Maybe:
selectedTestNamespace -> mutationEnabledTestNamespace
testNamespace - > mutationDisabledTestNamespace

**Comment by SchSeba** [comment]
/hold

gatekeeper is no deploy

`FEATURES_ENVIRONMENT=deploy FEATURES="sctp performance xt_u32 vrf container-mount-namespace" hack/feature-deploy.sh`

**Comment by SchSeba** [comment]
Hi @marzianor9 please rebase this PR

**Pull request merged**